modules/cms/classes/ComponentManager.php

```diff
@@ -187,32 +187,40 @@ public function hasComponent($name)
 
     /**
      * Makes a component object with properties set.
+     *
      * @param string $name A component class name or code.
      * @param CmsObject $cmsObject The Cms object that spawned this component.
      * @param array $properties The properties set by the Page or Layout.
+     * @param bool $isSoftComponent Defines if this is a soft component.
+     *
      * @return ComponentBase The component object.
+     *
```

Comment:
I think we have the @throws immediately following the @return elsewhere in the codebase, we should do that here too.

Reply:
I'd argue that spacing them out makes it easier to categorise and read. Thoughts?

Reply:
Sure, but I'm going off of what we already have in the codebase. We could consider a separate PR later to adjust that formatting across the codebase but for now we should stick with what's already being used.